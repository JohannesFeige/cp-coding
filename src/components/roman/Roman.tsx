import React from 'react';
import RomanNumeral from './RomanNumeral';

const Roman: React.FC = () => {
  return (
    <div>
      <h3>To Roman Numerals</h3>
      <p>The input will be translated to Roman Numerals</p>
      <RomanNumeral />
    </div>
  );
};

export default Roman;
