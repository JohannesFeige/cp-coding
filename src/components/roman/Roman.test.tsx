import React from 'react';
import { shallow } from 'enzyme';
import Roman from './Roman';

describe('Roman', () => {
  it('should render', () => {
    const wrapper = shallow(<Roman />);
    expect(wrapper).toMatchSnapshot();
  });
});
