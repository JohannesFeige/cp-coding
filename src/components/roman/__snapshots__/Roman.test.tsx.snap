// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Roman should render 1`] = `
<div>
  <h3>
    To Roman Numerals
  </h3>
  <p>
    The input will be translated to Roman Numerals
  </p>
  <RomanNumeral />
</div>
`;
